README.MD

```diff
@@ -14,6 +14,23 @@ for comparison are included. The comparisons are based on scoring
 philosophies for three statistical data types including categorical,
 continuous, and probabilistic.
 
+Possible enhancements include the ability to trace attributes from the
+candidate and benchmark maps to the outputs for analysis purposes.
+Additionally, we plan to enable comparisons across a hierarchy of
+samples beyond the current map level samples. These sample levels
+include catalogs (possibly using [STAC](https://stacspec.org/en)) and
+sub-samples (zonal level metrics within maps).
+
+Completed functionality:
+
+- Homogenize candidate and benchmark rasters for comparisons.
+- Two-class and multi-class categorical comparison case for 2 or
```

Comment:
Can you add continuous value comparison to the completed for me?

Reply:
I had not intended to add this portion back, I eliminated this part of the README previously.